CMakeLists.txt

```diff
@@ -125,6 +125,26 @@ set(SCAP_WORKBENCH_LINK_LIBRARIES
     Qt5::Widgets Qt5::XmlPatterns
     ${OPENSCAP_LIBRARIES})
 
+# ---------- RPATHS for linking
+
+# see https://gitlab.kitware.com/cmake/community/wikis/doc/cmake/RPATH-handling
+
+if (APPLE)
+    # when building, don't use the install RPATH already
+    # (but later on when installing)
+    set(CMAKE_BUILD_WITH_INSTALL_RPATH ON)
+
+    set(CMAKE_INSTALL_RPATH ${CMAKE_INSTALL_FULL_LIBDIR})
+
+    # add the automatically determined parts of the RPATH
+    # which point to directories outside the build tree to the install RPATH
+    set(CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE)
```

Comment:
Why the need for this?
I guess all libraries needed are in build tree, or default path of system.

Reply:
@yuumasato this is needed so that we can use `libopenscap` and place it in `./scap-workbench.app/Contents/Frameworks/`.

Reply:
I think `libopenscap` will be found because `CMAKE_INSTALL_RPATH` was already set to `${CMAKE_INSTALL_FULL_LIBDIR}`.